recipe/meta.yaml

```diff
@@ -25,7 +24,7 @@ requirements:
     - param >=1.6.1,<2.0
     - numpy >=1.0
     - matplotlib >=2.1
-    - bokeh >=0.12.15,<=0.13.0
+    - bokeh >=0.12.15,<=1.0
```

Comment:
Is it expected to break with `bokeh 1.0.1`? Or should it be OK with patch and minor releases?

The reason I ask is because @rsignell-usgs always needs the latest version to be able to run the new features and this constraint seems too tight. Maybe we can do `<2.0a` or something like that.

Reply:
I do anticipate some potential incompatibilities with the next minor release (i.e. 1.1.0) so let's go with ``>=0.12.15,<1.1``.

Reply:
I think it would probably be safe to pin to `<1.1.0`. We know there is a major PR in bokeh that will likely affect a number of things in HoloViews when it lands.